docker.go

```diff
@@ -270,6 +283,17 @@ func commandLogin(login Login) *exec.Cmd {
 	)
 }
 
+func commandLoginOIDC(login Login, accessToken string) *exec.Cmd {
```

Comment:
add accesstoken to the func name, and oidc to the accesstoken varible name

Reply:
renamed